app/views/components/_topic_list.html.erb

```diff
@@ -6,6 +6,8 @@
   margin_bottom ||= false
   ga4_data ||= {}
 
+  ul_data_attributes = { "module": "ga4-link-tracker", "ga4-track-links-only": "" } if !ga4_data.empty?
```

Comment:
I'm going commit by commit on this, so it might be answered later. This commit is moving tracking inside the topic list component, which includes adding the tracker and the `track-links-only` option. Is this option newly required or should it have been moved from somewhere else?

Reply:
This is newly required. I think it should have been on the existing tracking, as the previous tracker was initialised on a `div` element. But when I look at the examples in James' PR https://github.com/alphagov/collections/pull/3302 - all the example pages he showed have had their HTML updated since. For example the first two pages in his example use the `browse card` now, and the second two pages redirect to new pages entirely. So it's possible that the tracking is only relevant to this PR now anyway.

Reply:
Great, thanks 👍